statchatexpress/app.js

```diff
@@ -2,13 +2,33 @@ import express from "express";
 import cors from "cors";
 import morgan from "morgan";
 import { sequelize } from "./database.js";
-import { User, Stats } from "./models/index.js";
+import { User, UserGame } from "./models/index.js";
+import session from "express-session";
 
 const app = express();
-
-app.use(cors());
+const userSecret = process.env.SESSION_SECRET;
+const maximumAge = parseInt(process.env.SESSION_MAX_AGE);
+app.use(
+  cors({
+    origin: "http://localhost:3000",
+    credentials: true,
+  })
+);
 app.use(express.json()); // Middleware for parsing JSON bodies from HTTP requests
 app.use(morgan("combined"));
+app.use(
+  session({
+    secret: userSecret,
+    saveUninitialized: true,
+    //this saveUninitialized line will save a session in the browser even if the user does not do anything like login
+    resave: false,
+    //setting resave to false tells the server to check if anything has changed on the session before saving, this saves resources by not saving empty changes
```

Comment:
nitpick: Usually we put comments above the code instead of below!

Reply:
i will put the comment above